### DISCUSSION TITLE
performance comparison

### DISCUSSION THREAD

**Discussion by yumo-mt** [mobxjs/mobx#3036] [Q&A]
mobx5 vs mobx4 ， Proxy and defineProperty performance comparison, which is better, and how much is the difference ， thanks。

**Comment by iChenLei** [comment]
[CHANGELOG.md](https://github.com/mobxjs/mobx/blob/main/packages/mobx/CHANGELOG.md#500) for `how much is the difference?`

[perf scripts](https://github.com/mobxjs/mobx/blob/main/packages/mobx/__tests__/perf/index.js) for  `Proxy and defineProperty performance comparison, which is better?`

Anyway, I think mobx v6 is better than v4 and v5.

-----------------

mobx v6 chinese documentation https://zh.mobx.js.org, PR welcome.
